vips/vip-440/bsctestnet.ts

```diff
@@ -0,0 +1,149 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { POOL_REGISTRY, VTREASURY, CHAINLINK_ORACLE, RESILIENT_ORACLE } = NETWORK_ADDRESSES["sepolia"];
+
+export const COMPTROLLER = "0x7Aa39ab4BcA897F403425C9C6FDbd0f882Be0D70";
+export const BAL = "0xa18a6F0F51ddA0BAE3f2368bEE4b1542f6BE66C0";
+export const vBAL = "0xD4B82B7B7CdedB029e0E58AC1B6a04F6616BEd40";
+
+export const CONVERSION_INCENTIVE = 3e15;
```

Comment:
The incentive on Ethereum should be 0.03% (3e14). My fault. As it was set in the [VIP-430](https://app.venus.io/#/governance/proposal/430?chainId=56)